src/msw/graphicsd2d.cpp

```diff
@@ -4572,14 +4568,6 @@ void wxD2DContext::Flush()
     GetRenderTarget()->SetTransform(&currTransform);
 }
 
-void wxD2DContext::GetDPI(wxDouble* dpiX, wxDouble* dpiY)
-{
-    FLOAT x, y;
-    GetRenderTarget()->GetDpi(&x, &y);
-    if (dpiX != NULL) *dpiX = x;
-    if (dpiY != NULL) *dpiY = y;
-}
-
```

Comment:
Has this been removed because it's the same as `wxWindow::GetDPI()` used by the base class?

Reply:
The base version uses the window, which is DPI-Aware. This call always returns the system-DPI.

Maybe I can do something like:
```c++
// call base version that uses window
wxGraphicsContext::GetDPI(...)
if (x == 0 || y ==0) { /* renderer specific version */) }
```
This way Cairo can also return 72.

Reply:
Yes, returning hardcoded 72 never sat well with me neither. But OTOH this is another one of not perfectly backwards compatible changes I grew to be wary about... So maybe we should rather add `bool TryGetDPI(x, y)` and keep returning 72 from `GetDPI()` by default?